docs/content/developer/tutorials/sponsored-transactions/production-gas-station.mdx

```diff
@@ -0,0 +1,106 @@
+---
+Title: IOTA Gas Station Server
+Description: Learn how to use the IOTA Gas Station server to sponsor transactions on the IOTA network.
+Tags: [move-sc, tutorial, rust, rust]
+---
+
+# IOTA Gas Station server
+
+In this section, we will learn how to use IOTA's Gas Station Server to sponsor transactions on the IOTA network. We will use it to call the `free_trial` function of the `sponsored_transactions_packages` package.
+
+## Prerequisites
+
+Before we start, we should have the Gas Station server running. Ensure you have the following installed:
+* [Git](https://github.com/git-guides/install-git)
+* [Docker](https://docs.docker.com/engine/install/)
+* [Docker Compose](https://docs.docker.com/compose/install/)
+* [Rust](https://www.rust-lang.org/tools/install)
+
+## Setup the Gas Station server
+
+1. **Clone the IOTA Gas Station Repository:**
+
+```sh
+git clone https://github.com/iotaledger/gas-station
+```
+
+2. **Navigate to the Docker Directory and Generate the Config File:**
+
+```sh
+cd gas-station/docker
+../utils/./gas-station-tool.sh generate-sample-config --config-path config.yaml --docker-compose -n testnet
+```
+
+   **Note:** If the generated private key pair doesn’t meet your requirements, replace it with your own keys.
+
+3. **Set Up Authentication:** Define a bearer token for the Gas Station API using the `GAS_STATION_AUTH` environment variable.
+
+4. **Start the Gas Station**
+
+```sh
+GAS_STATION_AUTH=[bearer_token] docker-compose up
+```
```

Comment:
Please create and import a file in this doc and https://docs.iota.org/operator/gas-station/deployment/ so as to have one set of instructions only.

Reply:
Or add a link to said doc